src/main/java/com/team973/frc2025/subsystems/composables/DriveWithLimelight.java

```diff
@@ -25,6 +26,64 @@ public class DriveWithLimelight extends DriveComposable {
 
   private static final double TARGET_ANGLE_TOLERANCE_DEG = 6.0;
 
+  public static final double CORAL_MAX_VELOCITY_CONSTRAINTS = 1.6;
+  public static final double CORAL_MAX_ACCELERATION_CONSTRAINTS = 0.4;
+
+  public static final double NET_MAX_VELOCITY_CONSTRAINTS = 1.6;
+  public static final double NET_MAX_ACCELERATION_CONSTRAINTS = 0.1;
+  // elevator high consits of L4 and algae high
+  public static final double ELV_HIGH_MAX_VELOCITY_CONSTRAINTS = 1.6;
+  public static final double ELV_HIGH_MAX_ACCELERATION_CONSTRAINTS = 0.4;
+  // elevator low consits of L1-3 and algae
+  public static final double ELV_LOW_MAX_VELOCITY_CONSTRAINTS = 2.0;
+  public static final double ELV_LOW_MAX_ACCELERATION_CONSTRAINTS = 1.0;
+
+  public static TrapezoidProfile.Constraints m_coralXConstraint =
```

Comment:
We never really got a chance to finish debugging this stuff at mad town --- I'd like to eventually merge the constraints stuff but I'd _really_ like to merge the other stuff as soon as possible

Could you pull the DriveWithLimelight changes out of this PR so we can merge the rest? Then we can come back to the constraints stuff later